README.md

```diff
@@ -45,6 +45,8 @@ To close the container in the container command line run:
 exit
 ```
 
+You can also use this image for [Visual Studio remote containers](./RemoteContainers.md)
```

Comment:
[In VSCode docs](https://code.visualstudio.com/docs/remote/create-dev-container), the feature is referred to as `development containers`. Let's use the same terminology for consistency.

Reply:
Good, I think I fixed this now